wiki/Tournaments/ATS/2018/en.md

```diff
@@ -0,0 +1,167 @@
+----
+tags:
+- ATS 2018
+- ATS2018
+---
+Asian Taiko Showdown 2018
+=============================
+
+![Asian Taiko Showdown 2018](banner.png)
+
+> Unite the Diversity between us!
+
+**ATS 2018**, short for **Asian Taiko Showdown 2018**, is a double elimination one-on-one osu!taiko tournament hosted by [![][flag_ID] fajar13k](https://osu.ppy.sh/users/7100002). Previously ATS was known as [Indonesian Taiko Showdown (IDTS)](https://osu.ppy.sh/community/forums/topics/576070). This is the 2nd installment of the tournament and extension from Indonesia region to Asia.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| ---: | :--- |
+| Registration Phase | 2018-04-21/2018-05-26 |
+| Drawings | 2018-05-27 (19:00 UTC +7) |
+| Group Stage | 2018-06-02/2018-06-03 |
+| Round of 16 | 2018-06-09/2018-06-10 |
+| Quarterfinals | 2018-04-16/2018-04-17 |
+| Semifinals | 2018-06-23/2018-06-24 |
+| Finals | 2018-06-30/2018-07-01 |
+| Grand Finals | 2018-07-08 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| --- | :--- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Special Profile Badge, Userpage Banner |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | Userpage Banner |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | Userpage Banner |
+
+*Rest prizes are to be decided*
+
+## Organization
+
+The Asian Taiko Showdown 2018 is run by various osu!taiko community member.
+
+| Position | Members |
+| ------------ | -------------- |
+| Organizer | [![][flag_ID] fajar13k](https://osu.ppy.sh/users/7100002), [![][flag_US] JDrago14](https://osu.ppy.sh/users/7690078) |
+| Map Selectors | [![][flag_ID] Guardistack-](https://osu.ppy.sh/users/1602428), [![][flag_MY] cdhsausageboy](https://osu.ppy.sh/users/2403621), [![][flag_ID] Lightning Wyvern](https://osu.ppy.sh/users/1533122)  |
+| Commentators | [![][flag_AU] Jaye](https://osu.ppy.sh/users/4841352), [![][flag_ID] Niko-nyan](https://osu.ppy.sh/users/906991) |
+| Streamers | *To be decided* |
+| Designers | [![][flag_PH] OsuMe65](https://osu.ppy.sh/users/852867), [![][flag_ID] fajar13k](https://osu.ppy.sh/users/7100002), [![][flag_CL] Valelup](https://osu.ppy.sh/users/5281416) |
+| Referees | [![][flag_HK] -GN Junior](https://osu.ppy.sh/users/3901754), [![][flag_CL] xfraczynho](https://osu.ppy.sh/users/2192669), [![][flag_CL] Valelup](https://osu.ppy.sh/users/5281416), [![][flag_CL] Asagaki](https://osu.ppy.sh/users/2632902), [![][flag_PH] Pochacco](https://osu.ppy.sh/users/2927742), [![][flag_ID] Alwaysyukaz](https://osu.ppy.sh/users/4999506), [![][flag_KR] MetalStream](https://osu.ppy.sh/users/165027) |
+| Statisticians | [![][flag_ID] Aozora-](https://osu.ppy.sh/users/6918206), [![][flag_US] JDrago14](https://osu.ppy.sh/users/7690078) |
+
+## Links
+
+- [Livestream](https://www.twitch.tv/idtslive)
+- [Registration Form](https://goo.gl/forms/Z40RvcISgnMKNwGf2)
+
+-----------------------
+
+## Participants
+
+Players will be displayed after the registration phase is completed.
```

Comment:
suggestion:
`*The participants will be listed once the registration phase is completed.`